{
  "bug_location": "Unknown",
  "severity": 3,
  "categories": [
    "Unknown"
  ],
  "root_cause": "N/A - Analysis failed",
  "issue_number": 14071,
  "title": "Log backup: failed to register task after online reload certification when TLS enable"
}